src/sudoku.js

```diff
@@ -171,64 +171,65 @@ export const getPuzzleLeaderboard = (puzzle) => {
 };
 
 const isCellInvalid = (puzzle, indexRow, indexCol, value) => {
-  for (let row = 0; row < 9; row++) {
-    if (
-      indexRow != row &&
-      puzzle[row][indexCol].value &&
-      puzzle[row][indexCol].value === value &&
-      !puzzle[row][indexCol].guess
-    ) {
-      return true;
-    }
-  }
-
-  for (let col = 0; col < 9; col++) {
-    if (
-      indexCol != col &&
-      puzzle[indexRow][col].value &&
-      puzzle[indexRow][col].value === value &&
-      !puzzle[indexRow][col].guess
-    ) {
-      return true;
+    for (let row = 0; row < 9; row++) {
+        if (
+            indexRow != row &&
```

Comment:
It can be useful to create a function for this.

Reply:
out of scope

Reply:
but you can open a PR